other/aris-detect.bcheck

```diff
@@ -0,0 +1,27 @@
+metadata:
+  language: v1
+  name: "Aris Login Found"
+  description: "Tests for an exposed Aris Login Page"
+  author: "Awais"
+  tags: "Aris", "informative"
+
+run for each:
+  potential_path = 
+    "#default/home"

```

Comment:
Fragments are handled client-side rather than server-side. Has this request been tested against an Aris endpoint?